fields/types/html/HtmlField.js

```diff
@@ -40,6 +41,8 @@ module.exports = Field.create({
 
 	initWysiwyg () {
 		if (!this.props.wysiwyg) return;
+		// We only have it in the browser if we have wysiwyg
+		tinymce = require('tinymce');
```

Comment:
actually, not sure if this needs it. It's extra work, that field is the only field using tinymce, and loading that field for the first time is async anyway.